src/alchemlyb/tests/parsing/test_amber.py

```diff
@@ -23,15 +23,20 @@ def invalid_file(request):
                          [filename
                           for leg in load_simplesolvated()['data'].values()
                           for filename in leg])
-def test_dHdl(filename,
-              names=('time', 'lambdas'),
-              shape=(500, 1)):
+def test_dHdl(filename, names=('time', 'lambdas'), shape=(500, 1)):
```

Comment:
Normally we don't touch lines for reformatting unless they raise PEP8 issues so that we reduce noise in the diff. 

But it's ok, leave it, now that you already changed it.

Reply:
Ok, sorry I didn't know that!